cmd/argo/commands/install.go

```diff
@@ -45,8 +49,133 @@ var installCmd = &cobra.Command{
 	Run:   install,
 }
 
+func getClusterAdmin(clientset *kubernetes.Clientset) (bool, error) {
+	clusterRoles := clientset.RbacV1beta1().ClusterRoles()
+	list, err := clusterRoles.List(metav1.ListOptions{})
```

Comment:
Instead of List(), can we switch to Get() and use the IsNotFound() helper found in "k8s.io/apimachinery/pkg/api/errors"